Dockerfile

```diff
@@ -13,26 +13,28 @@
 # limitations under the License.
 
 # Build the manager binary
-FROM golang:1.10.3 as builder
+FROM golang:1.13 as builder
```

Comment:
maybe we can pin the specific  version as we do in others CAPX

Reply:
```suggestion
FROM golang:1.13.7 as builder
```

Reply:
Addressed, thanks!